user_templates/printer.cfg

```diff
@@ -24,11 +24,15 @@
 
 # ---------------------------------------------------------------------- MACHINE AXIS ----> Select only one line for each axis
 ### X axis -------------------------------------------------------------------------------
-# [include config/hardware/axis/X/1.8deg.cfg]
-# [include config/hardware/axis/X/0.9deg.cfg]
+# [include config/hardware/axis/X/1.8deg_1M.cfg] # Single X motor (e.g. Voron stock builds)
+# [include config/hardware/axis/X/1.8deg_2M.cfg] # Dual X motor (e.g. AWD mod)
+# [include config/hardware/axis/X/0.9deg_1M.cfg] # Single X motor (e.g. Voron stock builds)
+# [include config/hardware/axis/X/0.9deg_2M.cfg] # Dual X motor (e.g. AWD mod)
 ### Y axis -------------------------------------------------------------------------------
-# [include config/hardware/axis/Y/1.8deg.cfg]
-# [include config/hardware/axis/Y/0.9deg.cfg]
+# [include config/hardware/axis/Y/1.8deg_1M.cfg] # Single Y motor (e.g. Voron stock builds)
+# [include config/hardware/axis/Y/1.8deg_2M.cfg] # Dual Y motor (e.g. AWD mod)
+# [include config/hardware/axis/Y/0.9deg_1M.cfg] # Single Y motor (e.g. Voron stock builds)
+# [include config/hardware/axis/Y/0.9deg_2M.cfg] # Dual Y motor (e.g. AWD mod)
```

Comment:
For each line, can you remove the word "Voron" here? This line is for standard 1 motor axis, whatever the printer brand or type.
Second line is for AWD (2 motors for X) and this is ok like this.

Reply:
removed it